idd/Energy+.idd.in

```diff
@@ -1734,6 +1734,125 @@ Site:GroundTemperature:Deep,
        \type real
        \default 16
 
+Site:GroundTemperature:Undisturbed:FiniteDifference,
+       \memo Undisturbed ground temperature object using a
+       \memo detailed finite difference 1-D model
+  A1,  \field Name
```

Comment:
\min-fields 7 (defaults are only processed up through the last field in min-fields)  Check the other new objects as well.